Engine/AlgorithmManager.cs

```diff
@@ -75,7 +75,17 @@ public string AlgorithmId
         /// </summary>
         public TimeSpan CurrentTimeStepElapsed
         {
-            get { return _currentTimeStepTime == DateTime.MinValue ? TimeSpan.Zero : DateTime.UtcNow - _currentTimeStepTime; }
+            get {
```

Comment:
nit - style within the rest of the project typically places braces on a new line

Reply:
I'll fix 👍